<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPenStyle</class>
 <widget class="QDialog" name="UserPenStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="rowCount">
        <number>2</number>
       </property>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>штрих</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>пропуск</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>2.2</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>7</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Добавить элемент</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Удалить Элемент</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>offset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="offset">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>addButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>addButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserPenStyle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserPenStyle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
